setup-environment-internal

```diff
@@ -247,11 +247,17 @@ ln -sf "${MANIFESTS}" "${OEROOT}"/layers/
 
 DISTRO_DIRNAME=$(echo "${DISTRO}" | sed 's#[.-]#_#g')
 
+LMP_VER=$(git --git-dir "${MANIFESTS}"/.git describe --tags --abbrev=0)
+LMP_VER=$(( $LMP_VER + 1 ))
```

Comment:
if you just cloned v89 and tried to build with this, it would point you to a non-existent v90 cache, right? I think the default (non-ci) logic should just be `${LMP_VER}`.

Reply:
fixed, now uses the current version `https://storage.googleapis.com/lmp-cache/v89-sstate-cache`

Reply:
We need the increment because we are build with the cache for the next version, so it points `https://storage.googleapis.com/lmp-cache/v90-sstate-cache` again

Reply:
i think you are mixing what you as a platform dev does with a customer and also the fact we didn't have a normal "n+1" cache set up when you started on friday. Think this way: A customer repo-sync's v89. When they build - they should get the v89 cache, not v90.

Now as a dev - if you want to start working on v90 - that's n+1, but you've also broken from the v89 manifest.

Reply:
As we discussed on slack, this bump can indeed create unexpected mismatches for customers building lmp-manifest locally (from their own factory), so better to either drop (and document how our own devs can leverage our public dev cache) or have an additional flag/variable for deciding when or not doing the bump (e.g. dev mode).

Reply:
added a new variable `LMP_VER_DEV` to control the version, this will use the next version (+1) when defined before the setup-environment runs